### ISSUE TITLE
No access to location, doesn't appear under Location Services

### ISSUE THREAD

**Issue by cass-z** [anthonya1999/GoodNight#79]
iPhone 4s, iOS 7.1.2

GoodNight seems to work fine so far; however, although my Location Services is on, it does not appear under Location Services so that I can give access to it and enable Color Changing (Location) mode. I double-checked my Bundle Identifiers and appGroupIDs, uninstalled and reinstalled the app, turned on Background App Refresh, reset Location & Privacy, and restarted my phone—still unresolved.

**Comment by mariokorte** [comment]
Can anyone with another iOS7 device confirm this? I currently only have access to iOS8and iOS9 devices.

**Comment by SirDap** [comment]
i've got an old iPhone 4s running iOS 6.1.3 ... Changed the project's `iOS Deployment Target` to 6.1 and successfully installed GoodNight, but it opens to a black screen.

Would i need to upgrade to iOS 7 in order to perform the test, or is GoodNight in principle compatible with iOS 6.1?

**Comment by mariokorte** [comment]
No, 6.x is not possible due to the fact that this app requires several 7.x and higher APIs. Sorry.

**Comment by SirDap** [comment]
Cool! Thanks for the response. i'll upgrade to 7.x then and give this guy a whirl ;)

**Comment by SirDap** [comment]
Dang. Apple's stopped signing iOS 7.1.2 and i don't have any SHSH blobs saved for this device. Sorry folks
